AWSiOSSDKv2.xcodeproj/project.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>PreviewsEnabled</key>
+	<false/>
+</dict>
+</plist>
```

Comment:
what's this for?

Reply:
https://stackoverflow.com/questions/49478/git-ignore-file-for-xcode-projects

you can add `xcsettings` to gitignore if this is something newly generated with the latest Xcode